Description: Fix GetProgramCounter for sw64.
Author: Changwei Miao <miaochangwei@uniontech.com>

---
Last-Update: 2022-07-14

--- abseil-0~20210324.2.orig/absl/debugging/internal/examine_stack.cc
+++ abseil-0~20210324.2/absl/debugging/internal/examine_stack.cc
@@ -82,6 +82,8 @@ void* GetProgramCounter(void* vuc) {
       return reinterpret_cast<void*>(context->uc_mcontext.gregs[16]);
 #elif defined(__e2k__)
     return reinterpret_cast<void*>(context->uc_mcontext.cr0_hi);
+#elif defined(__sw_64__)
+    return reinterpret_cast<void*>(context->uc_mcontext.sc_pc);
 #else
 #error "Undefined Architecture."
 #endif
